### PR TITLE
Add documentation for PercentileKLL and its variants

### PR DESCRIPTION
**Pull request by cbalci**
Function implemented in: https://github.com/apache/pinot/pull/10643

### CODE REVIEW

**Comment by cbalci** [comment]
@navina please take a look.

File: configuration-reference/functions/percentilekll.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+All of the variants of `PercentileKLL` also support raw sketches in Pinot columns. This means you can create KLL Doubles sketches outside of Pinot and ingest them into columns as binary strings. `PercentileKLL` will identify these columns merge them to produce aggregate results.
+
+## Signature
+
+> PercentileKLL(column, percentile, kValue) -> Double
+
+* `column` (required): Name of the column to aggregate on. If the column is a multi value column, use `PERCENTILEKLLMV` variant.
+* `percentile` (required): Percentile value to be calculated [0..100]
+* `kValue`: Integer value which determines the size of the sketch. Default value is `200` which corresponds to a normalized rank error of about 1.65%. For defails please see the [accuracy vs size chart](https://datasketches.apache.org/docs/KLL/KLLAccuracyAndSize.html).
```
Comment (navina) [inline configuration-reference/functions/percentilekll.md]: nit: s/defails/details/ in all 4 `.md` files

**Review by navina — APPROVED** [review]
tks for the documentation ! +1 

cc: @Jackie-Jiang

File: configuration-reference/functions/percentilekll.md (hunk 1 above)
Comment (cbalci) [reply configuration-reference/functions/percentilekll.md]: Good catch, thanks! Fixed.

**Pull request merged**